crates/title_bar/src/title_bar.rs

```diff
@@ -186,36 +189,35 @@ impl Render for TitleBar {
-            h_flex()
-                .gap_1()
-                .map(|title_bar| {
-                    let mut render_project_items = title_bar_settings.show_branch_name
-                        || title_bar_settings.show_project_items;
-                    title_bar
-                        .when_some(
-                            self.application_menu.clone().filter(|_| !show_menus),
-                            |title_bar, menu| {
-                                render_project_items &=
-                                    !menu.update(cx, |menu, cx| menu.all_menus_shown(cx));
-                                title_bar.child(menu)
-                            },
-                        )
-                        .when(render_project_items, |title_bar| {
-                            title_bar
-                                .when(title_bar_settings.show_project_items, |title_bar| {
-                                    title_bar
-                                        .children(self.render_project_host(cx))
-                                        .child(self.render_project_name(cx))
-                                })
-                                .when(title_bar_settings.show_branch_name, |title_bar| {
-                                    title_bar.children(self.render_project_branch(cx))
-                                })
-                        })
-                })
-                .on_mouse_down(MouseButton::Left, |_, _, cx| cx.stop_propagation())
-                .into_any_element(),
-        );
+        children.push({
+            let el = h_flex().gap_1().map(|title_bar| {
+                let mut render_project_items =
+                    title_bar_settings.show_branch_name || title_bar_settings.show_project_items;
+                title_bar
+                    .when_some(
+                        self.application_menu.clone().filter(|_| !show_menus),
+                        |title_bar, menu| {
+                            render_project_items &=
+                                !menu.update(cx, |menu, cx| menu.all_menus_shown(cx));
+                            title_bar.child(menu)
+                        },
+                    )
+                    .when(render_project_items, |title_bar| {
+                        title_bar
+                            .when(title_bar_settings.show_project_items, |title_bar| {
+                                title_bar
+                                    .children(self.render_project_host(cx))
+                                    .child(self.render_project_name(cx))
+                            })
+                            .when(title_bar_settings.show_branch_name, |title_bar| {
+                                title_bar.children(self.render_project_branch(cx))
+                            })
+                    })
+            });
+            #[cfg(not(target_os = "macos"))]
+            let el = el.on_mouse_down(MouseButton::Left, |_, _, cx| cx.stop_propagation());
```

Comment:
Instead of adding this block and everything, can we instead do a `.when(cfg!(not(target_os = "macos")), ....)`?